core/src/client/entryPoints.js

```diff
@@ -14,5 +14,11 @@ module.exports = {
         title: '@labkey/components',
         permission: 'admin',
         path: './src/client/LabKeyUIComponentsPage'
+    },{
+        name: 'errorHandler',
+        title: 'Error Handler',
+        permission: 'admin',
+        path: './src/client/ErrorHandler',
+        generateLib: true
```

Comment:
could add a quick comment here saying where this lib is used:
```suggestion
        generateLib: true // used in errorView.jsp
```

Reply:
added